db/schema.rb

```diff
@@ -60,7 +60,7 @@
     t.string "surname", limit: 100
     t.string "banking_name"
     t.string "building_society_roll_number"
-    t.uuid "payment_id"
+    t.uuid "remove_column_payment_id"
```

Comment:
@slorek I was being cautious to keep this column in case something goes horribly wrong, the rename is to force so it's not accidentally used by the code, and just data. Comment to document this...
https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/capt-854/db/migrate/20230110135748_create_claims_payments.rb#L13-L14